Profiling NHSDigital-MessageDefinition

KazeemHamzat opened the pull request:
Profiling NHSDigital-MessageDefinition and update of NHSE dependency to UKCore to 0.0.5 pre-release

KazeemHamzat committed:
Profiling NHSDigital-MessageHeader

Profiling NHSDigital-MessageHeader and updating dependency to UKCore to 0.0.5 pre-release

KazeemHamzat committed:
Update to Profile Description

Update to Profile Description

KazeemHamzat committed:
Revert "Update to Profile Description"

This reverts commit c4930a371da888710667778ea22a8064d297ca56.

APageNHS commented on ValueSet/NHSDigital-message-events.xml:
```diff
@@ -1,7 +1,7 @@
-    <url value="https://fhir.nhs.uk/ValueSet/England-message-events" />
+    <url value="https://fhir.nhs.uk/ValueSet/England-MessageEvents" />
     <version value="1.0.12" />
     <name value="NHSMessageEvents" />
```
Doesn't this need to change to EnglandMessageEvents?

KazeemHamzat replied:
The format in both UKCore & NHSE Dev. Guide is: https://fhir.hl7.org.uk/ValueSet/UKCore-[BusinessNames]
However, we have two NHS message events, and they are program inclined, BARS and EPS. So I am proposing to change their Meta's to indicate the originating programmes. Therefore, the canonical URL`s for ValueSets and CodeSystems will change as follows: 
1.https://fhir.nhs.uk/ValueSet/England-MessageEventsBARS
2.https://fhir.nhs.uk/ValueSet/England-MessageEventsEPS
I will let you know once it's done so that you can update [UKCoreMessageEvent](https://simplifier.net/hl7fhirukcorer4/valueset-ukcore-messageevent)

KazeemHamzat committed:
Update NHSDigital-MessageDefinition.StructureDefinition.xml

Testing if validator is ok.

KazeemHamzat committed:
Update NHSDigital-MessageDefinition.StructureDefinition.xml

Testing validator issues on comments and markdown tags.

KazeemHamzat committed:
Merge branch 'Feature-IOPS-1570-NHSDigital-MessageDefinition' of https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide into Feature-IOPS-1570-NHSDigital-MessageDefinition

KazeemHamzat committed:
Fixing Comment Issues and Splitting CodeSystem

Fixing Comment Issues and Splitting CodeSystem

KazeemHamzat committed:
Fixing-CodeSystems-BARS

Fixing-CodeSystems-BARS

KazeemHamzat committed:
Fixing Canonical URL`s for CodeSystem-MessageEventsEPS

Fixing Canonical URL`s for CodeSystem-MessageEventsEPS: https://fhir.nhs.uk/CodeSystem/MessageEventEPS

KazeemHamzat committed:
Updating Canonical URL for CodeSyste-MessageEvents

Updating to: https://fhir.nhs.uk/CodeSystem/England-MessageEventEPS

KazeemHamzat committed:
Update2: Canonical URL for CodeSyste-MessageEvents

Updating Canonical URL for CodeSyste-MessageEvents

KazeemHamzat committed:
Updating Valueset-MessageEvents

Updating Valueset-MessageEvents

APageNHS commented on CodeSystem/BARS-message-events.json:
I'm confused on this file - it looks to almost match the xml one beloe, with the exception of the fact its json, and the url - why have we got 2 versions?

KazeemHamzat replied:
CodeSystem/BARS-message-events.json will be deleted. CodeSystem/CodeSystem-England-MessageEventBARS.xml was created to conform to NHSE Dev. Guide of files extension must be in XML.

APageNHS commented on MessageDefinition/MessageDefinition-dispense-notification-update.xml:
Not a major issue, but you could pretty print this in notepad++ to swap the "></tag> to "/>, in line with the other files

KazeemHamzat replied:
Resolved with https://www.samltool.com/prettyprint.php

APageNHS commented on MessageDefinition/MessageDefinition-dispense-notification.xml:
Not a major issue, but you could pretty print this in notepad++ to swap the "></tag> to "/>, in line with the other files

KazeemHamzat replied:
Resolved with https://www.samltool.com/prettyprint.php

APageNHS commented on Examples/SDS-Endpoint-Example.xml:
Not a major issue, but you could pretty print this in notepad++ to swap the "> to "/>, in line with the other files

KazeemHamzat replied:
Resolved with https://www.samltool.com/prettyprint.php

APageNHS commented on StructureDefinition/NHSDigital-MessageHeader.StructureDefinition.xml:
```diff
@@ -1,9 +1,25 @@
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the UKCore FHIR resource [UKCore MessageHeader ](https://simplifier.net/hl7fhirukcorer4/ukcore-messageheader)" />
```
Shouldn't cause a problem, but you have an extra space in the [ ] of the markdown link

UKCore FHIR, in text, should be referred to as FHIR UK Core

APageNHS commented on fhirpkg.lock.json:
Might be worth checking with someone re the fhirpkg.lock.json - not sure if that is needed, might be something that should be in a .gitignore? Or might be a byproduct of changing the package.json, which we don't do in UKCore and its an expected file to need to commit?

KazeemHamzat committed:
Merge branch 'develop' into Feature-IOPS-1570-NHSDigital-MessageDefinition

KazeemHamzat committed:
Fixing the issues raised with PR

Profile descriptions and Pretty nit XML

KazeemHamzat committed:
Update MessageDefinition-dispense-notification.xml

Updaate CodeSystem Canonical URL MessageEvents: https://fhir.nhs.uk/CodeSystem/England-MessageEventEPS

APageNHS reviewed (dismissed):
Comments are resolved, changes in line with expectations.

KazeemHamzat committed:
Delete fhirpkg.lock.json

Delete fhirpkg.lock.json

KazeemHamzat commented:
No longer to merge into DSTU2_develop_old
